lib/dpl/provider/pages.rb

```diff
@@ -53,13 +47,20 @@ def initialize(context, options)
         @deployment_file = !!options[:deployment_file]
 
         @gh_ref = "#{@gh_url}/#{slug}.git"
-        @gh_remote_url = "https://#{@gh_token}@#{@gh_ref}"
         @git_push_opts = @keep_history ? '' : ' --force'
         @git_commit_opts = (@allow_empty_commit and @keep_history) ? ' --allow-empty' : ''
 
         print_step "The repo is configured to use committer user and email." if @committer_from_gh
       end
 
+      def gh_token
```

Comment:
Hi @BanzaiMan, 

We have maybe found an issue on gh-pages deployment.

Since version `dpl-1.9.1.travis.2611.5` our [deployment](https://travis-ci.org/containous/traefik/builds/347267764#L1602) on gh-pages is no more working. 
We have this error message `gh-token must be provided for Pages provider to work.`

We have done no change on our travis configuration. And the last green deployment was with [version](https://travis-ci.org/containous/traefik/builds/346774791#L1711)  `dpl-1.8.48.travis.2486.5`

It could be relative to this change

Reply:
Please follow https://github.com/travis-ci/dpl/issues/762 for updates.